data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1023 @@
+			`	"The Free Worlds will also extend our protection, of course," the Chancellor adds.`
+			`	Teeneep responds, "I cannot sign that treaty yet. My people won't be satisfied until the kidnapped Hai are returned."`
+			`	The President replies, "We heard back from Syndicate Internal Affairs that they know the exact location of three staging facilities for a group called the 'Devil-Run Gang.' That group is behind the kidnappings." Turning to you, the President continues, "Please return to <destination>, and speak with Deputy Director Moray again. He may have a solution."`
+			`	Teeneep concludes the discussion. "I will travel with Captain <last> and see this matter through. I will expect your fleets and armies to be ready to assist us when the time is right." She turns to leave.`
+			choice
+				`	(Return to your ship.)`
+			`	On the way out, Teeneep tells you her next move. "The humans are so eager to raid the facilities that they haven't considered the need for information. We will have to arrange for someone to be 'captured' and brought to Philosopher's Paradise. The human authorities must delay their strike until then.`
+			`	"I'll need an escort back to <destination>. We'll need help from one more group, but I'll contact them on the way. You'll be paid, of course." She hands you <payment> for her most recent journey.`
+				accept
+	on complete
+		payment 400000
+		conversation
+			`You arrive at an office building with unusually high ceilings. When you enter, you notice one of the Quarg scientists you spoke to before is present, their height explaining the chosen meeting location. This must be the "other group" Teeneep said would help. You and Teeneep join the Quarg and Stephen Moray at a tall table.`
+			`	Teeneep explains her "intentional kidnapping" plan to the others, and Stephen says, "Believe it or not, I met a Hai recently that might be able to help. His name is Tutitot Eeegin, and he's expressed interest in helping us solve this problem. He's clever, and he has some experience acting." Stephen excuses himself to contact Tutitot while the Quarg turns to you and Teeneep.`
+			`	The Quarg in question begins in a low voice. "Teeneep here has asked if the Quarg have something you could use that would let Tutitot send telepathic signals millions of your kilometers. Alas, the Quarg have been unable to create such technology, and neither have the Korath." He smiles. "But the Quarg have been here for eons. In our time, we have discovered... 'devices,' beyond our ability to scientifically explain. One of these would be of use to you." He pulls out a glass orb, and it floats to the table; there seems to be a black worm with purple hairs inside. The orb has a blinking blue light on top.`
```

Comment:
If the idea is that this person would be taken to Deep Space 7C, that's not millions of kilometres, that's something like 200000 light years.
Also, is this stuff like what the Remnant have?
Can we... not have this here?
Also, I don't believe that the Quarg would do this. Certainly, not like this.